references/cli/def.go

```diff
@@ -1122,3 +1123,26 @@ func NewDefinitionGenAPICommand(c common.Args) *cobra.Command {
 	cmd.Flags().StringVar(&prefix, "prefix", "", "Specify the prefix of the generated Go struct.")
 	return cmd
 }
+
+// NewDefinitionCheckSystemCommand create the `vela def check-system` command to help check system definitions
+func NewDefinitionCheckSystemCommand(c common.Args) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check-system",
```

Comment:
any better name? Check seems to be read-only operations, but you're updating the resources.

Reply:
Why added a new command? We could add a stage to take the user to approve to update the definition.